src/android/wxapi/EntryActivity.java

```diff
@@ -7,6 +7,8 @@
 import android.os.Bundle;
 import android.util.Log;
 
+import com.getcapacitor.BridgeActivity;
```

Comment:
Used to get the MainActivity from `com.smartly.hybrid/MainActivity.java`, but we're only using the Class, which is `BridgeActivity`, i.e. not an instance of MainActivity. Hoping we can just swap that out here.